### DISCUSSION TITLE
What is buck1curr value?

### DISCUSSION THREAD

**Discussion by ombonbon** [johanmeijer/grott#251] [Q&A] [answered]
Hallo,

I have grott up and running in a proxmox container. Data is transferred to an MQTT server and is used by OpenHAB. I see below data is sent:
`{"device": "TUJ4BM702V", "time": "2022-12-06T16:50:59", "buffered": "no", "values": {"pvstatus": 8, "vpv1": 3366, "vpv2": 0, "ppv1": 2260, "ppv2": 0, "buck1curr": 43, "buck2curr": 0, "op_watt": 4380, "pvpowerout": 4380, "op_va": 1185280, "acchr_watt": 0, "acchr_VA": 0, "bat_Volt": 5156, "batterySoc": 50, "bus_volt": 4348, "grid_volt": 2294, "line_freq": 5002, "outputvolt": 2294, "pvgridvoltage": 2294, "outputfreq": 5019, "invtemp": 311, "dcdctemp": 287, "loadpercent": 125, "buck1_ntc": 279, "buck2_ntc": 0, "OP_Curr": 19, "Inv_Curr": 19, "AC_InWatt": 2550, "AC_InVA": 2521, "faultBit": 0, "warningBit": 0, "faultValue": 0, "warningValue": 0, "constantPowerOK": 2394, "epvtoday": 69, "pvenergytoday": 69, "epvtotal": 17169, "eacCharToday": 5, "eacCharTotal": 631, "ebatDischarToday": 0, "ebatDischarTotal": 4037, "eacDischarToday": 94, "eacDischarTotal": 15085, "ACCharCurr": 0, "ACDischarWatt": 4350, "ACDischarVA": 4630, "BatDischarWatt": 0, "BatDischarVA": 0, "BatWatt": -250}}`
My question is: what is the meaning of "buck1curr"? Sorry for this noob's question...

**Comment by johanmeijer** [comment, marked as answer]
If I look at internet (and although I am an electrical engineer, I am not and expert in solar inverters as well) is a buck converter a DC to DC converter. Apparently your inverter has two. Probably one per solar panel string (pp1 and pp2).   

Buck1curr is the current in DC-DC converter 1 (I think).

**Reply by ombonbon** [reply]
Ach so... Thank you very much @johanmeijer for your answer. Also, Thank you so much for creating this Grott.
